spec/lib/miq_automation_engine/service_methods/miq_ae_service_manageiq-providers-ansible_tower-configuration_manager-playbook_spec.rb

```diff
@@ -0,0 +1,12 @@
+module MiqAeServiceManageIQProvidersAnsibleTowerConfigurationManagerPlaybookSpec
```

Comment:
Comment was lost...
This `module` and `include` shouldn't be necessary, just the test.